acceptance/tests/facts/validate_file_system_size_bytes.rb

```diff
@@ -0,0 +1,39 @@
+# This test verifies that the numbers for file systems sizes are positive from "facter" and "puppet facts"
+# This is a regression test for FACT-1578
+test_name "C100110: verify that file system sizes are positive" do
+  tag 'risk:high'
+
+  require 'json'
+
+  agents.each do |agent|
+    step("verify that facter returns positive numbers for the mount points byte fields") do
+
+      on(agent, facter("--json")) do |facter_output|
+        facter_results = JSON.parse(facter_output.stdout)
+        facter_results['mountpoints'].each_key do |mount_key|
```

Comment:
That's good to know, but in this case I can't do that as it will only fail if we happen to have a number that overflows the conversion correctly. Its doesn't fail for all the values.